static/js/portico/landing-page.js

```diff
@@ -63,12 +63,14 @@ var apps_events = function () {
         android: {
             image: "/static/images/app-screenshots/zulip-android.png",
             alt: "Android",
+            show_instructions: false,
             description: "Zulip's native Android app makes it easy to keep up while on the go.",
             link: "https://play.google.com/store/apps/details?id=com.zulipmobile",
         },
         ios: {
             image: "/static/images/app-screenshots/zulip-iphone-rough.png",
             alt: "iOS",
+            show_instructions: false,
```

Comment:
I don't understand this change -- it does not seem to do anything.  Why do you claim this fixes something?

Reply:
@timabbott  show_instruction property show download instruction for desktop app, but i observed a weird behaviour, when i clicks on ios or android icon it shows download instruction(which is for desktop apps) in paragraph, , as a solution to this weird behaviour i setted show_instruction property to false for ios, android and corrects the behaviour . you can see [weird behaviour(left) and correct behaviour(right)](https://user-images.githubusercontent.com/36422396/55293095-f8fbb980-540f-11e9-870f-466840d41626.gif), **what are your thoughts?**